docs/zh_cn/supported_models/supported_models.md

```diff
@@ -81,6 +81,7 @@
 |          DeepSeek-MoE          |     16B     | LLM  |    Yes    |   No    |   No    |  No  |  No   |
 |          DeepSeek-V2           |  16B, 236B  | LLM  |    Yes    |   No    |   No    |  No  |  No   |
 |         DeepSeek-V2.5          |    236B     | LLM  |    Yes    |   No    |   No    |  No  |  No   |
+|          DeepSeek-VL2          |  3B - 27B   | MLLM |    Yes    |   No    |   No    |  No  |  No   |
```

Comment:
We'd better add a document about how to quickly setup env and use deepseek-vl2, just like
https://github.com/InternLM/lmdeploy/blob/main/docs/en/multi_modal/cogvlm.md